src/DssSpell.sol

```diff
@@ -74,190 +120,324 @@ contract DssSpellAction is DssAction {
 
-        // GFX Labs - 29.76 MKR - 0x9b68c14e936104e9a7a24c712beecdc220002984
-        MKR.transfer(GFXLABS_WALLET,          29.76 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
+        // Bandhar     -  7.68 MKR - 0xE83B6a503A94a5b764CCF00667689B3a522ABc21
+        MKR.transfer(BANDHAR,         7.68 ether); // note: ether is a keyword helper, only MKR is transferred here
 
-        // QGov - 29.76 MKR - 0xB0524D8707F76c681901b782372EbeD2d4bA28a6
-        MKR.transfer(QGOV_WALLET,             29.76 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
+        // Libertas    -  7.04 MKR - 0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f
+        MKR.transfer(LIBERTAS,        7.04 ether); // note: ether is a keyword helper, only MKR is transferred here
 
-        // TRUE NAME - 29.76 MKR - 0x612f7924c367575a0edf21333d96b15f1b345a5d
-        MKR.transfer(TRUENAME_WALLET,         29.76 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
+        // PALC        -  2.24 MKR - 0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A
+        MKR.transfer(PALC,            2.24 ether); // note: ether is a keyword helper, only MKR is transferred here
 
-        // UPMaker - 9.92 MKR - 0xbb819df169670dc71a16f58f55956fe642cc6bcd
-        MKR.transfer(UPMAKER_WALLET,          9.92 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
+        // Harmony     -  1.92 MKR - 0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2
+        MKR.transfer(HARMONY,         1.92 ether); // note: ether is a keyword helper, only MKR is transferred here
 
-        // WBC - 9.92 MKR - 0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47
-        MKR.transfer(WBC_WALLET,              9.92 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
+        // VoteWizard  -  1.6  MKR - 0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1
+        MKR.transfer(VOTEWIZARD,      1.6  ether); // note: ether is a keyword helper, only MKR is transferred here
 
-        // Libertas - 9.92 MKR - 0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f
-        MKR.transfer(LIBERTAS_WALLET,         9.92 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
+        // Navigator   -  0.32 MKR - 0x11406a9CC2e37425F15f920F494A51133ac93072
+        MKR.transfer(NAVIGATOR,       0.32 ether); // note: ether is a keyword helper, only MKR is transferred here
 
-        // Codeknight - 9.92 MKR - 0xf6006d4cF95d6CB2CD1E24AC215D5BF3bca81e7D
-        MKR.transfer(CODEKNIGHT_WALLET,       9.92 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
+        // ----- CRVV1ETHSTETH-A 1st Stage Offboarding -----
+        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-3/21238/6
+
+        // Set CRVV1ETHSTETH-A Debt Ceiling to 0
+        DssExecLib.setIlkDebtCeiling("CRVV1ETHSTETH-A", 0);
```

Comment:
From my full review:

_Decrease Global Debt Ceiling by Total Amount of Offboarded Ilks `line` Cached_
          This is not happening, per the exec *sheet* cell F4: "Removing CRVV1ETHSTETH-A from autoline - do not change as plenty of buffer, discussed GovOps 2023-07-04". Have raised with governance facilitators that this needs to be documented in the exec doc as otherwise (in absence of the exec sheet, only referring to doc) it appears that the spell team has forgotten an action.
          Raised with governance facilitators, will keep discussing with them until there is a conclusion.
          This is blocking, content either needs to follow pattern (with no deviation) in spell code *or* the deviation needs to be noted in the exec doc.

_____

Exec doc is being updated to support this as a permissible action by the spell team and to clarify that the spell team did not forget the content regarding global Line reduction (which would be 100M in this case). This decision was supposedly made during GovOps call and an updated exec doc provides provenance backing it which the spell team can then action via the spell code.